.emacs.d/test/lisp/install.el

```diff
@@ -4,4 +4,7 @@
 (package-initialize)
 (package-refresh-contents)
 
+;; add undercover for coverage reporting
+(setf kotct/dependency-list (cons 'undercover kotct/dependency-list))
```

Comment:
So undercover is installed, but only on Travis (via install script instead of on init).